pkg/ccl/streamingccl/settings.go

```diff
@@ -119,3 +119,12 @@ var DumpFrontierEntries = settings.RegisterDurationSetting(
 	0,
 	settings.NonNegativeDuration,
 )
+
```

Comment:
could you reword the commit message? It seems to imply that span config replication is disabled by default.

Reply:
indeed, doing now